src/reference/antora/modules/ROOT/pages/mail.adoc

```diff
@@ -367,7 +811,52 @@ public interface SearchTermStrategy {
 
 The following example relies on `TestSearchTermStrategy` rather than the default `SearchTermStrategy`:
 
-[source,xml]
+[tabs]
+======
+Java::
++
+[source,java,role="primary"]
+----
+@Bean
+public ImapMailReceiver imapMailReceiver(SearchTermStrategy searchTermStrategy) {
+    ImapMailReceiver receiver = new ImapMailReceiver("imap:something");
+    // ...
+    receiver.setSearchTermStrategy(searchTermStrategy);
+    return receiver;
+}
+
+@Bean
+SearchTermStrategy searchTermStrategy() {
+    return new TestSearchTermStrategy();
+}
+----
+Kotlin::
++
+[source,kotlin,role="secondary"]
+----
+@Bean
+fun imapMailReceiver(): ImapMailReceiver {
+    return ImapMailReceiver("imap:something").apply {
+        // ...
+        setSearchTermStrategy(TestSearchTermStrategy())
+    }
+}
+----
+Groovy::
++
+[source,groovy,role="secondary"]
+----
+@Bean
+ImapMailReceiver imapMailReceiver(SearchTermStrategy searchTermStrategy) {
+    def receiver = new ImapMailReceiver("imap:something")
+    // ...
+    receiver.searchTermStrategy = TestSearchTermStrategy()
```

Comment:
I think this is a typo: we are supposed to use an injection in the method argument.